client/src/main/scala/org/apache/celeborn/client/CommitManager.scala

```diff
@@ -291,4 +292,27 @@ class CommitManager(appId: String, val conf: CelebornConf, lifecycleManager: Lif
     }
     (totalWritten, totalFileCount)
   }
+
+  def handleShutdownWorker(shutdownWorker: WorkerInfo): Unit = {
```

Comment:
During a worker shutdown, it won't revive new reserve slot request and push data request.
So here in the ShufflePartitionLocationInfo, there won't contains new partition location of this worker.
Since if we don't commit, it won't restart.